### PR TITLE
Undeprecates gafferpy

### LINKED ISSUE #982: Remove gafferpy deprecation notice
Instead of fishbowl replacing gafferpy for Gaffer 2.0, it has been decided that it will instead be used to maintain gafferpy https://github.com/gchq/gaffer-tools/issues/981.
Remove the deprecation notice in the README.
This reverts https://github.com/gchq/gaffer-tools/issues/951.

### PR DESCRIPTION
**Pull request by t92549**
This reverts commit 94a4d38f0ffc30c132783e43a93cebb9c2e97ee5.

# Related Issue

- Resolve #982

### CODE REVIEW

File: python-shell/README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
-# Deprecated
+# README
+
+Copyright 2016 Crown Copyright
```
Comment (t11947) [inline python-shell/README.md:3]: Is the plan not to update the header as we're only reinstating things not strictly changing?

Comment (t92549) [reply python-shell/README.md:3]: Yes this PR is strictly a git revert, but you're right that date does need changing. I also don't like the fact the readme starts with a license when there is a separate license file.

Comment (GCHQDeveloper314) [reply python-shell/README.md:3]: It feels like we need to rework the way the license stuff is managed more generally. Do these kind of files really need a header? For example, we don't have a header on every page of the docs.

**Review by GCHQDeveloper314 — APPROVED** [review]

File: python-shell/README.md (hunk 1 above)
Comment (t92549) [reply python-shell/README.md:3]: Yeah I am not sure, seems unnecessary and makes it harder to read for a readme.

**Pull request merged**